lang/python/python-pytest-xdist/Makefile

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (C) 2020 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-pytest-xdist
+PKG_VERSION:=1.34.0
+PKG_RELEASE:=1
+
+PYPI_NAME:=pytest-xdist
+PKG_HASH:=340e8e83e2a4c0d861bdd8d05c5d7b7143f6eea0aba902997db15c2a86be04ee
+
+PKG_MAINTAINER:=Jan Pavlinec <jan.pavlinec@nic.cz>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_BUILD_PARALLEL:=0
+HOST_PYTHON3_PACKAGE_BUILD_DEPENDS:=setuptools-scm
+
+include ../pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-pytest-xdist
+  SUBMENU:=Python
+  SECTION:=lang
+  CATEGORY:=Languages
+  TITLE:=Pytest plugin for parallel test running
+  URL:=https://github.com/pytest-dev/pytest-xdist
+  DEPENDS:= \
+	+python3-light \
+	+python3-pytest \
+	+python3-execnet \
+	+python3-apipkg \
+	+python3-multiprocessing
```

Comment:
pytest-xdist [does not depend](https://github.com/pytest-dev/pytest-xdist/blob/v1.34.0/setup.py#L3) on apipkg directly. It also depends on pytest-forked and six.

Also, it [uses multiprocessing](https://github.com/pytest-dev/pytest-xdist/blob/v1.34.0/src/xdist/plugin.py#L15-L18) only when it cannot import `os.cpu_count` (because `os.cpu_count` was [added](https://docs.python.org/3/library/os.html#os.cpu_count) in Python 3.4 but it supports older versions). Since our Python always has `os.cpu_count`, multiprocessing is not necessary.

Reply:
Removed python3-multiprocessing and python3-apipkg 
Added python3-pytest-forked python3-six